lib/models/Signs.js

```diff
@@ -0,0 +1,31 @@
+const pool = require('../utils/pool');
+
+module.exports = class Sign {
+  id;
+  word;
+  aka;
+  url;
+  introduced;
+  proficiency;
+
```

Comment:
Don't forget our model filename will be singular with a capitalized first letter (Sign.js).